Himotoki/decode.swift

```diff
@@ -33,3 +43,8 @@ public func decodeDictionary<T: Decodable where T.DecodedType == T>(object: AnyO
         return accum
     }
 }
+
+/// - Throws: DecodeError
+public func decodeDictionary<T: Decodable where T.DecodedType == T>(object: AnyObject, rootKeyPath: KeyPath) throws -> [String: T] {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 132 characters